src/java/org/apache/cassandra/db/compaction/CompactionStrategyManager.java

```diff
@@ -301,6 +303,7 @@ public void pause()
         writeLock.lock();
         try
         {
+            logger.debug("Bereng paused\n" + ExceptionUtils.getStackTrace(new Exception()));
```

Comment:
same

Reply:
Yep they are in their own isolated commit to be dropped annotated with 'DO NOT MERGE'